src/components/canvas/Box.jsx

```diff
@@ -2,34 +2,28 @@ import { useFrame } from '@react-three/fiber'
 import { useRouter } from 'next/router'
 import { useRef, useState } from 'react'
 
-const BoxComponent = ({ route }) => {
+export default function BoxComponent({ route }) {
   const router = useRouter()
-  // This reference will give us direct access to the THREE.Mesh object
   const mesh = useRef(null)
-  // Set up state for the hovered and active state
   const [hovered, setHover] = useState(false)
-  // Subscribe this component to the render-loop, rotate the mesh every frame
-  useFrame((state, delta) =>
-    mesh.current
-      ? (mesh.current.rotation.y = mesh.current.rotation.x += 0.01)
-      : null
-  )
-  // Return the view, these are regular Threejs elements expressed in JSX
+
+  useFrame((state, delta) => {
+    mesh.current.rotation.y = mesh.current.rotation.x += delta
```

Comment:
why is it that the check for the mesh to be non null was deleted? is it not needed?

Reply:
references are guaranteed inside effects/useframe. if you do this: 

```jsx
const ref = useRef<THREE.Mesh>(null!)
useEffect(() => {
  console.log(ref.current)
}, [])
return <mesh ref={ref} />
```

then it won't cause a type error.